huxley/core/middlewares.py

```diff
@@ -1,35 +1,46 @@
-# Copyright (c) 2011-2015 Berkeley Model United Nations. All rights reserved.
+# Copyright (c) 2011-2017 Berkeley Model United Nations. All rights reserved.
 # Use of this source code is governed by a BSD License (see LICENSE).
 
 import json, logging, sys, traceback
 
 
 class ExceptionLoggerMiddleware(object):
```

Comment:
This is probably a good opportunity to move this middleware file from `core` to `logging`, since this is all logging middleware. Also, let's follow the convention of naming it `middleware` singular. Remember to update the location in settings as well!